pkgs/development/python-modules/python-utils/default.nix

```diff
@@ -40,6 +41,10 @@ buildPythonPackage rec {
     pytestCheckHook
   ] ++ passthru.optional-dependencies.loguru;
 
+  propagatedBuildInputs = [
+    typing-extensions
+  ];
```

Comment:
Can you move that above the optional-dependencies? Makes more sense to have it there, because both are dependencies.